lib/mocaccino.js

```diff
@@ -13,6 +13,7 @@ var through = require('through2');
 var resolve = require('resolve');
 var listen  = require('listen');
 var supportsColor = require('supports-color');
+var mochaReporters = require('mocha').reporters;
```

Comment:
This makes it necessary to move mocha from the devDependencies to the dependencies.